README.md

```diff
@@ -164,6 +164,18 @@ class Document
 end
 ```
 
+#### Note on datetime type
+
+By default datetime fields are persisted as UNIX timestamps with milisecond precission in DynamoDB. If you prefer datetimes to be stored as ISO-8601 formatted strings instead then set `convert_date_to_string` to `true`
+
+```ruby
+class Document
+  include DynamoId::Document
+
+  field :sent_at, :datetime, convert_date_to_string: true
```

Comment:
We already have similar option for boolean field `store_as_native_boolean`

```ruby
field :active, :boolean, store_as_native_boolean: true
```

It makes sense to name new option in similar way

Reply:
Again, it may be useful to apply this new option globally and override it for specific field

Reply:
We should follow existing naming patterns. 👍